README.md

```diff
@@ -22,7 +22,7 @@ npm install pcln-design-system
 If you'd like to contribute to the design system, we'd love to have your help.
 If you work at Priceline.com, please join the #design-system Slack channel and let us know what you'd like to work on.
 
-For all contributors, please be sure to read the [Contributing](docs/Contributing.md) doc.
+For all contributors, please be sure to read the [Contributing](docs/pages/Contributing.md) doc.
```

Comment:
@bertya, thanks for your 1st contribution!

Seems that [a suggestion](https://github.com/pricelinelabs/design-system/issues/390#issuecomment-427847467) in #390 is advising to moving the `Contributing.md` to the root. Would you mind updating?